doc/getting_and_using_mir.md

```diff
@@ -1,157 +1,127 @@
 
 On Ubuntu the latest Mir release is available from the Mir team's "release PPA".
 We recommend always using the latest release, but if you don't wish to use the
-PPA, there will be an earlier release of Mir in the Ubuntu archive, but some
-things may not work as described here.
+PPA, there will be also a release of Mir in the Ubuntu archive, but some things
+may not work as described here.
 
 To add the PPA to your system:
 
-    $ sudo add-apt-repository ppa:mir-team/release
-    $ sudo apt update
-
-To remove the PPA from your system:
-
-    $ sudo ppa-purge mir-team/release
+    sudo add-apt-repository --update ppa:mir-team/release
 
 You can install the Mir examples along with the Mir graphics drivers as follows:
 
-    $ sudo apt install mir-demos qterminal qtwayland5
-    $ sudo apt install mir-graphics-drivers-desktop
-
-## Getting Mir examples on Fedora
+    sudo apt install mir-demos mir-graphics-drivers-desktop
 
-On Fedora Mir is available from the archive. We recommend using qterminal with
-Mir as Qt currently works better with Mir's Wayland support than other options.   
+It is also useful to install Wayland support for Qt and qterminal:
 
-You can install the Mir examples as follows:
-
-    $ sudo dnf install mir-demos qterminal
-
-Using Mir examples
-------------------
-
-For convenient testing under X11 there's a "miral-app" script that wraps the
-commands used to start a server and then launches a terminal (as the current
-user):
+    sudo apt install qterminal qtwayland5
 
-    $ miral-app
-
-To run independently of X11 you need to grant access to the graphics hardware
-(by running as root) and specify a VT to run in. There's a "miral-desktop"
-script that wraps to start the server (as root) and then launch a terminal
-(as the current user):
+To remove the PPA from your system:
 
-    $ miral-desktop
-    
-For more options see *Options when running the Mir example shell* below.
-    
-### Running applications on Mir
+    sudo ppa-purge mir-team/release
 
-__On Ubuntu__, if you use the command-line launched by `miral-app` or 
-`miral-desktop` native Mir applications (which include native Mir clients
-and those that use SDL or the GTK+, Qt toolkits) can be started as usual:
+## Getting Mir on Fedora
 
-    $ sudo apt install kate neverputt 
-    $ mir_demo_client_chain_jumping_buffers
-    $ kate
-    $ neverputt
-    $ gedit
+On Fedora Mir is available from the archive:
 
-__On Fedora__, we can use the Wayland backends for SDL or Qt:
+    sudo dnf install mir-demos
 
-    $ sudo dnf install kate neverball-neverputt
-    $ mir_demo_client_chain_jumping_buffers
-    $ kate
-    $ neverputt
+It is also useful to install qterminal:
 
-From outside the MirAL session applications can be run using the miral-run script:
+    sudo dnf install qterminal
 
-    $ miral-run kate
-    $ miral-run neverputt
+## Getting Mir on Arch
 
-### Running for X11 applications
+If you're on Arch Linux, you can install 'mir' from the AUR.
```

Comment:
I know this was directly taken from my suggestion on IRC, but I think `On Arch Linux, you can install  the [mir](https://aur.archlinux.org/packages/mir/) package from the AUR.` would be a bit clearer.